modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -1157,10 +1180,43 @@ paths:
                       format: int64
                       type: integer
             responses:
-                204:
+                200:
                     content:
-                        application/json: {}
-                        application/xml: {}
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/UserGroup"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/UserGroup"
+                    description:
+                        default response
+            tags: ["UserGroup"]
+        put:
+            description:
+                Associates the user group with the given external reference code to the asset library.
```

Comment:
we are not using the ERC here

"Associates the user group with the asset library."

Reply:
Fixed.